l1-contracts/src/core/libraries/ConstantsGen.sol

```diff
@@ -16,6 +16,7 @@ library Constants {
   uint256 internal constant MAX_FIELD_VALUE =
     21888242871839275222246405745257275088548364400416034343698204186575808495616;
   uint256 internal constant ARGS_LENGTH = 16;
+  uint256 internal constant MAX_ARGS_TO_ALL_ENQUEUED_CALLS = 16000;
```

Comment:
Worth adding a "do not change without consulting AVM"?

Is it clear these are fields?